### PR TITLE
feat: pomodoro-feature-added

### PR DESCRIPTION
**Pull request by Megha-Joshi**
In this PR, I have added -
- Pomodoro feature with focus and break time.

### CODE REVIEW

File: src/pages/Homepage/homepage.jsx [hunk 1]
```diff
@@ -12,34 +12,34 @@ const [currTask, setCurrTask] = useState({});
...
+}
 }
 const customStyle = {
-    overlay: {
-      top: "6rem",
-      backgroundColor: "rgba(52, 58, 64, 0.8)",
-    },
-    content: {
-      width: "18rem",
-    //   height: "20rem",
```
Comment (zoyatanweer) [inline src/pages/Homepage/homepage.jsx:38]: remove commented code

File: src/pages/Homepage/homepage.jsx [hunk 2]
```diff
@@ -12,34 +12,34 @@ const [currTask, setCurrTask] = useState({});
...
-      backgroundColor: "var(--box-color)",
-    },
-  };
+overlay: {
+top: "6rem",
+backgroundColor: "rgba(52, 58, 64, 0.8)",
+},
+content: {
+width: "18rem",
+// height: "20rem",
```
Comment (zoyatanweer) [inline src/pages/Homepage/homepage.jsx:38]: remove commented code if not required.

File: src/pages/Homepage/homepage.jsx [hunk 3]
```diff
@@ -12,34 +12,34 @@ const [currTask, setCurrTask] = useState({});
...
-    content: {
-      width: "18rem",
-    //   height: "20rem",
-      margin: "5rem auto",
-      backgroundColor: "var(--box-color)",
-    },
-  };
+overlay: {
+top: "6rem",
+backgroundColor: "rgba(52, 58, 64, 0.8)",
```
Comment (zoyatanweer) [inline src/pages/Homepage/homepage.jsx:34]: use var for color

**Review by zoyatanweer — APPROVED** [review]
Everything looks good to me. Code is nice and understandable. Do the suggested changes and you're good to merge.

**Review by srishtigautam21 — APPROVED** [review]
The code is clean
Good to merge

File: src/pages/Homepage/homepage.jsx (hunk 2 above)
Comment (Megha-Joshi) [reply src/pages/Homepage/homepage.jsx:38]: Ok..I will do it.

File: src/pages/Homepage/homepage.jsx (hunk 3 above)
Comment (Megha-Joshi) [reply src/pages/Homepage/homepage.jsx:34]: This color is only used here.

File: src/pages/Homepage/homepage.jsx (hunk 2 above)
Comment (Megha-Joshi) [reply src/pages/Homepage/homepage.jsx:38]: Ok..I will do it.

**Pull request merged**